downstream/modules/platform/ref-single-eda-controller-with-internal-db.adoc

```diff
@@ -27,17 +23,9 @@ automationedacontroller_pg_database='automationedacontroller'
 automationedacontroller_pg_username='automationedacontroller'
 automationedacontroller_pg_password='<password>'
 
-# The full routable URL used by EDA to connect to a controller host.
-# This URL is required if there is no Automation Controller configured
-# in inventory.
-#
 automation_controller_main_url = 'https://controller.example.com/'
```

Comment:
```suggestion
# The full routable URL used by EDA to connect to a controller host.
# This URL is required if there is no Automation Controller configured
# in inventory.
#
automation_controller_main_url = 'https://controller.example.com/'

# Boolean flag used to verify Automation Controller's
# web certificates when making calls from Automation EDA Controller.
#
automationedacontroller_controller_verify_ssl = true
```

Reply:
@jamesmarshall24 - I assumed you meant that I should remove this entire line. Let me know if you intended something different.

Reply:
I have updated this PR based on suggested changes.

Reply:
@jself-sudoku the way git shows diffs can be confusing, but the `automation_controller_main_url` should still be there just moved into the section under the comments. If you look at the green in my suggested change, you'll still see it there. Can you re-do the PR with that line added back?

It also looks like we're missing this line too at the top:
```
[automationedacontroller]
automationedacontroller.example.com ansible_connection=local
```

Reply:
@jamesmarshall24 - can you use the **Files changed** tab to show where these lines should be? 

Here's what's confusing . I used the exact example that displays in the published information and just added the `[all: vars]` above automationedacontroller_admin_password='<eda-password>'

If there are missing lines, they've been missing since the beginning. At this point, I'm not sure where you're saying to add the lines that you say are missing now. Should it be the following (missing text in **bold**)? 

# Automation EDA Controller Configuration
#

**[automationedacontroller]
automationedacontroller.example.com ansible_connection=local**

[all:vars]
automationedacontroller_admin_password='<eda-password>'

automationedacontroller_pg_host=''
automationedacontroller_pg_port=5432

automationedacontroller_pg_database='automationedacontroller'
automationedacontroller_pg_username='automationedacontroller'
automationedacontroller_pg_password='<password>'

Reply:
I think we also need this still at the bottom:
```
# This URL is required if there is no Automation Controller configured
# in inventory.
#
automation_controller_main_url = 'https://controller.example.com/'

# Boolean flag used to verify Automation Controller's
# web certificates when making calls from Automation EDA Controller.
#
automationedacontroller_controller_verify_ssl = true
```